assets/src/edit-story/elements/shape/display.js

```diff
@@ -30,10 +30,12 @@ const Element = styled.div`
   ${elementWithBackgroundColor}
 `;
 
-function ShapeDisplay({ element: { backgroundColor } }) {
-  const props = {
-    backgroundColor,
-  };
+function ShapeDisplay({ element: { isDefaultBackground, backgroundColor } }) {
+  const props = isDefaultBackground
+    ? null
+    : {
+        backgroundColor,
+      };
   return <Element {...props} />;
```

Comment:
This would be a bit cleaner with just:

```suggestion
  if ( isDefaultBackground ) {
     return <Element />;
  }
  return <Element backgroundColor={backgroundColor} />;
```

Reply:
Changed in 00284b7